src/components/popover/popover.jsx

```diff
@@ -93,16 +83,17 @@ Popover.propTypes = {
   children: PropTypes.node,
   flush: PropTypes.oneOf(['left', 'right']),
   onClose: PropTypes.func,
-  startOpen: PropTypes.bool,
+  /** Handles either a FocusEvent or MouseEvent. This is an additional predicate for configuring the closing behavior. */
+  shouldClose: PropTypes.func,
```

Comment:
I'm bad at naming, and this is a bit pedantic, but whenever I see questions as an argument to something, I expect it to a boolean. I get why this is a function - its a function _that returns_ a boolean - but I wonder if there is a better name hidden somewhere? Nothing comes to mind for me so maybe not..

Reply:
Yeah, I tried looking up best practice around predicates and couldn't find much. 

I figured React-ers might be familiar with the `should*`, `will*`, `did*` name patterns given React class methods. I was hoping for a cooler `use*` approach but didn't come up with anything then .